### ISSUE TITLE
ValueConvertible cannot be used with dictionary literal in swift 3.0.2

### ISSUE THREAD

**Issue by acecilia** [OpenKitten/MongoKitten#104]
Hello:
Just started using MongoKitten and I got to this error:
![captura de pantalla 2017-04-04 a las 0 42 59](https://cloud.githubusercontent.com/assets/7938671/24634744/c7b53a68-18cf-11e7-8050-15ea5916c94e.png)

I am running swift 3.0.2 on macOSX 10.11.6 (EL Capitan).

Thanks!

**Comment by Joannis** [comment]
MongoKitten 3 requires any dictionary to be explicitly marked as a Document.
So that would work like this:

```swift
let document: Document = [
    "key": ["$exists": true] as Document
]
```

This is solved in MongoKitten 4 (but requires Swift 3.1)

**Comment by acecilia** [comment]
Nice, thanks for the answer ;)
